src/main/java/com/softserveinc/task03/Country.java

```diff
@@ -0,0 +1,97 @@
+package com.softserveinc.task03;
+
+import com.softserveinc.task02.City;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Country {
+    private String name;
+    private String continent;
+    private String code;
+    private City capital;
+    private List<City> cities;
+
+    public Country() {
+
```

Comment:
You have to initialize the field cities

Reply:
It is initialized in the constructor.
try to use simple array

Reply:
This is also a constructor without parameters. Because you don't initialize the field cities the test fails.
Add code `cities = new ArraysList<>()`